src/main/java/com/ss/ita/rozetka/pageobject/elements/filters/Filter.java

```diff
@@ -0,0 +1,167 @@
+package com.ss.ita.rozetka.pageobject.elements.filters;
+
+import com.codeborne.selenide.ElementsCollection;
+import com.codeborne.selenide.SelenideElement;
+import io.qameta.allure.Step;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static com.codeborne.selenide.CollectionCondition.sizeGreaterThan;
+import static com.codeborne.selenide.CollectionCondition.sizeLessThan;
+import static com.codeborne.selenide.Condition.*;
+import static com.codeborne.selenide.Selenide.$$x;
+import static com.codeborne.selenide.Selenide.$x;
+import static java.lang.String.format;
+
+/**
+ * Represents a single filter block with options and, if exists, a search field.<br>
+ * <b>Can be created only from FilterStack.</b>
+ */
+public class Filter {
```

Comment:
will this be used anywhere else except in FilterStack class?
if not, merge it with FilterStack